handouts/2020-02-04-diversity-blurb.txt

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
On a high-level, 

1. If you feel you're constrained for words, feel free to expand it to 300+ words. We can cut and slice according to the medium we're using the blurb for.
2. The paragraphs after paragraph 2 are pretty solid. 👍 
3. We don't really talk about what nilenso _does_. Let's add a paragraph about that as well so it can cater to folks who don't know what we do or are, in addition to our focus for the today. Let me know if you need help with resources on this.

The rest of the comments are inline.

Reply:
1. It's already above 300 words. ;) But I'll expand where necessary.
2. Cool.
3. I wasn't sure if this was necessary, given that it's not the focus of this particular blurb -- and I never run across someone who doesn't know what nilenso is anymore. ;) I can add a bit at the end.

Reply:
@deobald Yeah, I think it makes sense to add a paragraph covering nilenso as a company. An example blurb is in the Work For Steve document, you can take ideas off of that.

Reply:
I was told it's valuable to have all the comments here, so x-posting from Slack.

My preference would be for the hiring blurb to showcase us as a company
- we care about craft
- we take on challenging projects
- we care about impact
- we care about your learning/growth: conference sponsorship, attendance, talks, library
- we think a healthy work/life balance is important
- we have progressive policies: parental leave, menstrual leave
- we are nice people to work with
- we are an open and transparent organisation

We are making an effort to make the company diverse and inclusive, but that shouldn’t overshadow who we are. A hiring blurb is not the place to acknowledge systemic bias and provide references to reading material so people can educate themselves about it, imo. It would make for a great blog post, though.